### ISSUE TITLE
kaustinen: syncing node doesn't go through the transition

### ISSUE THREAD

**Issue by gballet** [gballet/go-ethereum#272]
Unlike the previous issue described in #271 , starting the syncing node before the transition manages to sync the network, but at the transition time, the elephant banner doesn't seem to work and the error message shows:

```
########## BAD BLOCK #########
Block: 8 (0x46e9ed87025685d26344c8d00805a89356a873cb6a6b39cde4a0258f92ebeefa)
Error: invalid merkle root (remote: 0000000000000000000000000000000000000000000000000000000000000000 local: 25acafa84c55ba32e8ac837fd7de8eb1d6712fcd9927151dc14352a058af36e2) dberr: %!w(<nil>)
Platform: geth (devel) go1.19.4 amd64 linux
VCS: 26ffffc5-20230912 (dirty)
Chain config: &params.ChainConfig{ChainID:69420, HomesteadBlock:0, DAOForkBlock:0, DAOForkSupport:false, EIP150Block:0, EIP155Block:0, EIP158Block:0, ByzantiumBlock:0, ConstantinopleBlock:0, PetersburgBlock:0, IstanbulBlock:0, MuirGlacierBlock:0, BerlinBlock:0, LondonBlock:0, ArrowGlacierBlock:<nil>, GrayGlacierBlock:<nil>, MergeNetsplitBlock:0, ShanghaiTime:(*uint64)(0xc000674bf8), CancunTime:(*uint64)(nil), PragueTime:(*uint64)(0xc000674c00), TerminalTotalDifficulty:0, TerminalTotalDifficultyPassed:true, Ethash:(*params.EthashConfig)(nil), Clique:(*params.CliqueConfig)(nil), IsDevMode:false}
Receipts:
##############################
```

It does seem that the original root is 0, which is a bug in itself. But the syncing node should go through the transition nonetheless.

**Comment by gballet** [comment]
Gajinder's instructions to reproduce the bug:

> pull spindevnets , and in [fork-scheduler.sh](http://fork-scheduler.sh/), set VERGE_EPOCH=0, it will set transition at block 1 without making EL genesis verge
basically will set pragueTime > genesis timestamp  but < network genesis time (actually 1 hour before network genesis time)

**Comment by gballet** [comment]
Fixed in #275
